UmbracoCms.8.1.1/Views/Authentication/_Login.cshtml

```diff
@@ -0,0 +1,23 @@
+﻿@inherits Umbraco.Web.Mvc.UmbracoViewPage<Acme.UI.Models.Authentication.LoginModel>
```

Comment:
I dont think you need these view files in the website do you? They are in your vs project